### PR TITLE
LPS-120862 Enable CT for Blogs

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-120862

Also enabled change tracking for ReadingTimeEntry since it is called by [BlogsReadingTimeEntryLocalServiceWrapper here](https://github.com/liferay/liferay-portal/blob/d146f6f8b1846555cef3f4ee7166e5c951ccdcd6/modules/apps/blogs/blogs-reading-time/src/main/java/com/liferay/blogs/reading/time/internal/service/BlogsReadingTimeEntryLocalServiceWrapper.java#L65) and was causing a nested operation `CTTransactionException`.

I have `isHidable() = true`  for BlogsStatsUserCTDisplayRenderer and ReadingTimeEntryCTDisplayRenderer since those seem like 'background clutter' changes to a user.  

I wasn't sure which columns to set `change-tracking-resolution-type` for in the service.xml's, so that might be wrong. 

Ran tests in `change-tracking-test`, `reading-time-test`, and `blogs-test`. `blogs-test` had failures, but they were the same failures as when I ran against master afterwards.

### CODE REVIEW

File: modules/apps/blogs/blogs-service/service.xml [hunk 1]
```diff
@@ -23,12 +23,12 @@
 
 		<!-- Other fields -->
 
-		<column name="title" type="String" uad-nonanonymizable="true" />
-		<column name="subtitle" type="String" uad-nonanonymizable="true" />
-		<column name="urlTitle" type="String" uad-nonanonymizable="true" />
-		<column name="description" type="String" uad-nonanonymizable="true" />
-		<column name="content" type="String" uad-nonanonymizable="true" />
-		<column name="displayDate" type="Date" />
+		<column change-tracking-resolution-type="max" name="title" type="String" uad-nonanonymizable="true" />
```
Comment (Preston-Crary) [inline modules/apps/blogs/blogs-service/service.xml:26]: All of these should be "merge" types.

File: modules/apps/blogs/blogs-service/service.xml [hunk 2]
```diff
@@ -195,8 +195,8 @@
 
 		<!-- Other fields -->
 
-		<column name="entryCount" type="int" />
-		<column name="lastPostDate" type="Date" />
+		<column change-tracking-resolution-type="max" name="entryCount" type="int" />
```
Comment (Preston-Crary) [inline modules/apps/blogs/blogs-service/service.xml:198]: We should leave entryCount alone unless we add in something to recalculate it before publishing. Let's leave it as strict for now.

File: modules/apps/reading-time/reading-time-service/service.xml [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 		<column name="classNameId" type="long" />
 		<column name="classPK" type="long" />
-		<column name="readingTime" type="long" />
+		<column change-tracking-resolution-type="max" name="readingTime" type="long" />
```
Comment (Preston-Crary) [inline modules/apps/reading-time/reading-time-service/service.xml:26]: I think this one should be merge, someone be making the article shorter.

**Comment by ctang3** [comment]
@Preston-Crary  updated with changes, thanks!

File: modules/apps/blogs/blogs-service/service.xml [hunk 4]
```diff
@@ -40,7 +40,7 @@
 		<column name="smallImageId" type="long" uad-nonanonymizable="true" />
 		<column name="smallImageURL" type="String" />
 		<column name="lastPublishDate" type="Date" />
-		<column name="status" type="int" />
+		<column change-tracking-resolution-type="max" name="status" type="int" />
```
Comment (Preston-Crary) [inline modules/apps/blogs/blogs-service/service.xml:43]: Let's leave this one out too. Status doesn't increment like lastPostDate does. It's a flag to determine the workflow and I think if it's different there may be other workflow rows that have different states as well.

Comment (ctang3) [reply modules/apps/blogs/blogs-service/service.xml]: Updated

**Comment by Preston-Crary** [comment]
After https://github.com/shuyangzhou/liferay-portal/pull/10384 is merged in please rebase this on top of that one and resend, thanks!

**Pull request closed without merging**